packages/client/config-overrides.js

```diff
@@ -0,0 +1,40 @@
+const { override } = require("customize-cra")
+const cspHtmlWebpackPlugin = require("csp-html-webpack-plugin")
+
+const cspConfigPolicy = {
+  "default-src": ["'none'"],
+  "script-src": [],
+  "style-src": ["'self'", "'unsafe-inline'"],
+  "img-src": ["'self'", "data:", "cid:", "mid:"],
+  "child-src": ["data:"],
+  "font-src": ["'self'"],
+  "media-src": ["'self'", "data:", "cid:", "mid:"],
+  "object-src": ["'none'"],
+  "form-action": ["'none'"]
+}
+
+const cspConfigPolicy_dev = {
+  ...cspConfigPolicy,
+  "connect-src": ["'self'", "ws://localhost:3000"],
```

Comment:
Create-react-app uses a websocket connection to automatically reload the browser when code changes are made. We want a content security policy (CSP) in development that allows this. But we want a more locked-down configuration in production.